package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dgraph-js-http",
-  "version": "21.03.1",
```

Comment:
Do we need a rc1 release or can we go directly to v23.0.0? Do you see any risks with that?

Reply:
As we discussed - there is little downside to going with an rc first.  We can push the full release later at any time.  We've also pushed rc candidates before: https://www.npmjs.com/package/dgraph-js-http?activeTab=versions